XamlStyler.Core/DocumentProcessors/CommentDocumentProcessor.cs

```diff
@@ -73,13 +71,12 @@ public void Process(XmlReader xmlReader, StringBuilder output, ElementProcessCon
             }
             else
             {
-                output
-                    .Append(currentIndentString)
-                    .Append("<!--")
-                    .Append(' ', this.options.CommentSpaces)
-                    .Append(content.Trim())
-                    .Append(' ', this.options.CommentSpaces)
-                    .Append("-->");
+                output.Append(currentIndentString)
+                      .Append("<!--")
+                      .Append(' ', this.options.CommentSpaces)
```

Comment:
A bunch of these changes look like they are not related to the new option. Can you revert these non-essential changes?